cmd/atlante/cmd/server.go

```diff
@@ -5,6 +5,9 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/go-spatial/maptoolkit/atlante/queuer"
+	"github.com/prometheus/common/log"
```

Comment:
intentional import of this logger?

Reply:
Yeah, I've been using this logger for atlante. Using the log.Infof and log.Warnf ... trying to see if this is a better way forward or not? I'm not sure, how I came to this one.